src/components/vars.sass

```diff
@@ -284,17 +284,19 @@ $learninglog-green-light-6: #cfe5d2
 $learninglog-green-light-7: #dcecde
 $learninglog-green-light-8: #edf5ee
 $learninglog-green-light-9: #f6faf7
-$classwork-purple-dark-1: #aa2ff0
-$classwork-purple: #c765ff
-$classwork-purple-light-1: #cc74ff
-$classwork-purple-light-2: #d68cff
-$classwork-purple-light-3: #dc9eff
-$classwork-purple-light-4: #e3b1ff
-$classwork-purple-light-5: #ecc9ff
-$classwork-purple-light-6: #f0d5ff
-$classwork-purple-light-7: #f4e0ff
-$classwork-purple-light-8: #f9efff
-$classwork-purple-light-9: #fcf7ff
+
+$violet-dark-1: #aa2ff0
+$violet: #c765ff
+$violet-light-1: #cc74ff
+$violet-light-2: #d68cff
+$violet-light-3: #dc9eff
+$violet-light-4: #e3b1ff
+$violet-light-5: #ecc9ff
+$violet-light-6: #f0d5ff
+$violet-light-7: #f4e0ff
+$violet-light-8: #f9efff
+$violet-light-9: #fcf7ff
+
```

Comment:
I'm not sure about this renaming. It makes sense in that the new names are closer to the names used in the specs, and that could reduce confusion. I don't know if it was considered important to include "classwork" in the old names, though. Obviously, there was a desire to associate them with the Class Work tab. Maybe names starting with `classwork-violet` would be a good compromise? @kswenson, it looks like you approved Matt's PR where he introduced the old names. Do you know/recall if it was deemed important to include the "classwork" part?

Reply:
Ah, I misunderstand a comment you made yesterday about updating something to match the spec and thought you meant updating the variable names to match the spec. I'll switch it back.